tests/trunc-double-uniform.ispc

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
double tests should be skipped on DG1/TGLLP

Reply:
It's skipped. I added in the next commit.